docs/quickstarts/csharp-quickstart.md

```diff
@@ -118,8 +163,22 @@ message HelloReply {
 ## Generate gRPC code
 
 Next we need to update the gRPC code used by our application to use the new service definition. 
-The Grpc.Tools NuGet package contains the protoc and protobuf C# plugin binaries you will need to generate the code. This example project already depends on the NuGet package `Grpc.Tools.1.0.0`, so it should be included in the `examples/csharp/helloworld/packages` when the `Greeter.sln` solution is built from your IDE. 
 
+The Grpc.Tools NuGet package contains the protoc and protobuf C# plugin binaries you will need to generate the code. 
+
+#### If you're following the Visual Studio story or restored packages with NuGet from the command line:
```

Comment:
This and the next header are the only usages of "story". Let's not use that word, since we aren't using it elsewhere. We should probably just have a "Using blah" header here and below instead.